frontend/app_flowy/packages/flowy_infra/lib/theme.dart

```diff
@@ -1,35 +1,27 @@
 import 'package:flowy_infra/colorscheme/colorscheme.dart';
 import 'package:flutter/material.dart';
 
-const List<String> builtInThemes = [
-  'light',
-];
+class BuiltInTheme {
+  static const String light = 'light';
+  static const String dandelion = 'dandelion';
+}
```

Comment:
What are the advantages of using a class over a list or enum? I initially wanted this to be used to check whether a theme is built in or not